clients/iscmove/iscmoveclient/client_anchor_ptb.go

```diff
@@ -64,7 +68,7 @@ func PTBTakeAndTransferCoinBalance(
 	argAssets := iotago.Argument{NestedResult: &iotago.NestedResult{Cmd: *argBorrow.Result, Result: 0}}
 	argB := iotago.Argument{NestedResult: &iotago.NestedResult{Cmd: *argBorrow.Result, Result: 1}}
 
-	for coinType, coinBalance := range assets.Coins {
+	for _, coinType := range slices.Sorted(slices.Values(maps.Keys(assets.Coins))) {
```

Comment:
why do we need to sort the coins here?

Reply:
It's to make sure that the resulting PTB is deterministic.
Probably it's not needed, but I'm not sure.